organize/models.py

```diff
@@ -182,7 +183,7 @@ class Coorganizer(models.Model):
         related_name="coorganizers")
     email = models.EmailField()
     first_name = models.CharField(max_length=30)
-    last_name = models.CharField(max_length=30)
+    last_name = models.CharField(max_length=30, blank=True)
```

Comment:
We have to remember to check if everything is right after adding blank here. We have methods taking last_name as something that should just exist because it was required.